src/backend/distributed/sql/udfs/worker_adjust_identity_column_seq_ranges/11.2-2.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE FUNCTION pg_catalog.worker_adjust_identity_column_seq_ranges(regclass)
+    RETURNS VOID
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$worker_adjust_identity_column_seq_ranges$$;
+COMMENT ON FUNCTION pg_catalog.worker_adjust_identity_column_seq_ranges(regclass)
+    IS 'modify identity column seq ranges to produce globally unique values';
```

Comment:
It is not really common that we have 2 versions of a function that are identical.

How about adding a comment and including the same file here?

```suggestion
-- Since we backported the UDF below from version 11.3, the definition is the same
#include "udfs/worker_adjust_identity_column_seq_ranges/11.3-1.sql"
```

Reply:
Looks great. I didn't know that we can use `#include` directive in a udf definition file.

Reply:
Hmm it looks like we can't use it since it fails on CI. See https://app.circleci.com/pipelines/github/citusdata/citus/31889/workflows/3181846e-1345-4073-af6c-f4f0116981d1/jobs/1076152?invite=true#step-102-312